src/Plugin/Queue/Workers/Kafka/KafkaWorker.php

```diff
@@ -28,6 +28,8 @@ class KafkaWorker implements WorkerRunnerInterface
 
 	private Client $client;
 	private string $brokerList;
+	/** @var array{string: string}  */
+	private array $customMapping;
```

Comment:
A bit confusing, it contains array of the following ["string": ....] ?

Reply:
yes `@var array{string: string}` means that structure of array should be `{"key":"value"}`

Reply:
proper syntax for `{"key":"value"}` is `array<string,string>` `array{string:string}` means that array will have key name "string" and contain string value

I guess in case we have mixed name in keys here we should fix it, otherwise if we have structure that contains only "string" as a key, we can leave it as is